tests/test_lm/test_integration/test_jsrun/test_2/cmds

```diff
@@ -1,3 +1,3 @@
-# Task needs to be assigned node 1, physical cores 1 and 2, HW thread 1 of each physical core
+# Task needs to be assigned node 1, physical core 1, HW thread 1, 1 GPU
```

Comment:
We should stick to one convention:

"Physical core 1" means "physical core with (or at) index 1"
"1 GPU" refers to quantity. I suggest changing this to "GPU 1" (or GPU 0 - whichever is appropriate)

Reply:
okay

Reply:
@vivek-bala, does the check look for a specific GPU index or the number of GPUs?